### PR TITLE
CI CD fixes 

### PR DESCRIPTION
**Pull request by namArora3112**
## Description
https://jira.corp.adobe.com/browse/MOB-22364

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by yangyansong-adbe — APPROVED** [review]

**Review by cacheung — APPROVED** [review]
We can merge this PR first, but can we take a look of Android sample app CI, is it still broken? It was disabled in old architecture app.

File: .circleci/config.yml [hunk 1]
```diff
@@ -62,25 +71,42 @@ jobs:
...
           name: Building iOS Sample App
-          command: cd apps/AEPSampleApp && npx react-native run-ios --verbose --terminal terminal
+          command: |
+            cd apps/AEPSampleAppNewArchEnabled/ios
+            xcodebuild \
+              -workspace AEPSampleAppNewArchEnabled.xcworkspace \
+              -scheme AEPSampleAppNewArchEnabled \
+              -sdk iphonesimulator \
+              -destination "generic/platform=iOS Simulator" \
+              clean build
```
Comment (cacheung) [inline .circleci/config.yml:109]: Is Android app still having error?

Comment (namArora3112) [reply .circleci/config.yml:109]: yes I have tried to run it on android but still facing the same

**Comment by namArora3112** [comment]
> We can merge this PR first, but can we take a look of Android sample app CI, is it still broken? It was disabled in old architecture app.

I gave it a shot and tried various approaches, but despite my efforts, I was unable to get it running on Android after trying numerous workarounds. This can be considered a separate task for further enhancements and investigation.

File: .circleci/config.yml (hunk 1 above)
Comment (cacheung) [reply .circleci/config.yml:109]: Could you update the sample app to AEPSampleAppNewArchEnabled in the comments in the Android at the bottom of the file?

Comment (namArora3112) [reply .circleci/config.yml:109]: sure

Comment (cacheung) [reply .circleci/config.yml:109]: > [CXX1210] /root/project/apps/**AEPSampleApp**/node_modules/react-native-reanimated/android/CMakeLists.txt debug|arm64-v8a : No compatible library found

**Pull request merged**